app/javascript/src/components/Workspace.tsx

```diff
@@ -54,9 +58,24 @@ export default class Workspace extends React.Component<WorkspaceProps> {
       this.isMouseDown && now.valueOf() - this.mouseDown.valueOf() > 150
   }
 
+  handleKeyPress = (event: any) => {
+    if ((event.ctrlKey || event.metaKey) && event.keyCode == 67)
```

Comment:
To save people the time it takes to look up keycodes, sometimes I like to write'em as consts:

```
const C_KEY = 67
const V_KEY = 86

...

event.keyCode = V_KEY
```

Probably just a style preference.